### PR TITLE
create README.md file 

### PR DESCRIPTION
**Pull request by sudoshweta**
create a standardized README.md template.

### CODE REVIEW

**Comment by rnburn** [comment]
I think @isaachier has a README.md as part of his PR.

**Comment by isaachier** [comment]
Ya thanks for the help but check out the existing PR and maybe we can add it after that is accepted.

File: README.md [hunk 1]
```diff
@@ -1 +1,22 @@
...
+
+## Required Reading
+
+In order to understand the C platform API, one must first be familiar with the
+[OpenTracing project](http://opentracing.io) and
+[terminology](http://opentracing.io/documentation/pages/spec) more generally.
+
+## C/C++98
+
+This library requires C++11 or later. But if you're interested in a C or C++98
```
Comment (isaachier) [inline README.md]: Actually this would not be the case here. Any C/C++ compiler should be able to do it.

Comment (sudoshweta) [reply README.md]: okay, I will review it once more.

**Comment by isaachier** [comment]
Please see other pull request. I don't think this one needs updating before that is accepted.

**Comment by sudoshweta** [comment]
okay.

**Comment by isaachier** [comment]
Now we can revisit this @skzinc.

**Comment by isaachier** [comment]
No updates so closing.

**Pull request closed without merging**